script.js

```diff
@@ -15,13 +17,224 @@ navPort.addEventListener('click', () => {
+    modalSection.removeChild(modalSection.firstChild);
+  }
+};
+
+const createModal = (arr, i) => {
+  const modalSection = document.querySelector('.modal-section');
+  const modal = document.createElement('div');
+  modal.className = 'modal';
+  const modalImage = document.createElement('div');
+  modalImage.className = 'modal-image';
+  const modalContent = document.createElement('div');
+  modalContent.className = 'modal-content';
+  const modalBtn = document.createElement('div');
+  modalBtn.className = 'modal-btn';
+  const modalImg = document.createElement('img');
+  modalImg.src = arr[i].modalCardImage;
+  const modalClose = document.createElement('span');
+  modalClose.className = 'modal-close';
+  const modalHead = document.createElement('h1');
+  const modalTags = document.createElement('div');
+  modalTags.className = 'tags';
+  const modalDescription = document.createElement('p');
+  const btn1 = document.createElement('a');
+  btn1.className = 'live';
+  const btn2 = document.createElement('a');
+  btn2.className = 'source';
+  const btn1Image = document.createElement('img');
+  btn1Image.src = './images/see-live-icon.svg';
+  const btn2Image = document.createElement('img');
+  btn2Image.src = './images/IconGitHub.png';
+  modalClose.innerHTML = '&times';
+  modalHead.innerHTML = arr[i].title;
+  const spanTag = Array.from(Array(arr[i].languageTags.length).keys());
+  spanTag.forEach((x) => {
+    modalTags.appendChild(document.createElement('span'));
+    const line = modalTags.children[x];
+    line.innerHTML = arr[i].languageTags[x];
+  });
+
+  modalDescription.innerHTML = arr[i].description;
+  btn1.innerHTML = 'See Live';
+  btn2.innerHTML = 'See Source';
+
+  modalSection.appendChild(modal);
+  modal.append(modalImage, modalContent, modalDescription, modalBtn);
+  modalImage.append(modalImg, modalClose);
+  modalContent.append(modalHead, modalTags);
+  // modalFlex.append()
+  modalBtn.append(btn1, btn2);
+  btn1.appendChild(btn1Image);
+  btn2.appendChild(btn2Image);
+
+  modalClose.addEventListener('click', () => {
+    modalSection.classList.toggle('active');
+    deleteModal();
+    fixed.classList.remove('fixed');
   });
-   
\ No newline at end of file
+};
+
+const cardButton = Array.from(document.querySelectorAll('.card-container .btn'));
+cardButton.forEach((i) => i.addEventListener('click', () => {
+  modalSection.classList.toggle('active');
+  createModal(CardArray, i.id);
+  fixed.classList.add('fixed');
+}));
+
+const form = document.getElementById('form');
+const email = document.getElementById('email');
+const emailError = document.querySelector('.emailError');
+const formInput = Array.from(document.querySelectorAll('.input'));
+
+form.addEventListener('submit', (e) => {
+  if (email.value !== email.value.toLowerCase()) {
+    e.preventDefault();
+    emailError.innerHTML = 'email should be lower case';
+    emailError.classList.add('display');
+    email.classList.add('invalid');
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly ensure the error message shows near the submit button or just under the submit button to meet the project requirement below: 

:point_down: 
![near_submit](https://user-images.githubusercontent.com/59397159/174133723-2bb022e3-14b0-4bef-947a-51121c72ed23.png)

*near or under the button*
![error_](https://user-images.githubusercontent.com/59397159/174133719-c6f897dd-5b0e-46bc-afd5-1f81b668428e.png)

Reply:
Okay, we will work on it.